src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -108,13 +111,28 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
       >
         Implement Change Request
       </NERButton>
-      <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
+      <Menu
+        open={dropdownOpen}
+        anchorEl={anchorEl}
+        onClose={handleDropdownClose}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'right'
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'right'
+        }}
+      >
         <MenuItem
           to={`${routes.PROJECTS_NEW}?crId=${changeRequest.crId}&wbs=${projectWbsPipe(changeRequest.wbsNum)}`}
           component={RouterLink}
           onClick={handleDropdownClose}
           disabled={!isUserAllowedToImplement}
         >
+          <ListItemIcon>
+            <CreateNewFolderIcon fontSize="small" />
+          </ListItemIcon>
           Create New Project
```

Comment:
You could put <ListItemText> around the text 'Create New Project'. This might make the code easier to read and align more with the documentation for the MUI Menu. You would have to do this everywhere there is text.

Reply:
I see what you're getting at, but I think it looks the same without the ListItemText tag, so let's leave it off here, just makes the file a bit less cluttered